README.md

```diff
@@ -13,9 +13,10 @@ The pre-release is at a **release candidate** level and we could use your help t
 ## Installation
 
 ```
-$ npm i --save @fortawesome/fontawesome
-$ npm i --save @fortawesome/fontawesome-free-solid
-$ npm i --save @fortawesome/vue-fontawesome
```

Comment:
Thanks for the PR! If you are going to remove the `$` (which I don't really care either way) but let's be consistent with the steps below for yarn. Also I think that we need to add the install for regular in the yarn section too.